transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -88,6 +75,7 @@
 
     private WritableByteChannel byteChannel;
 
+    @Deprecated
```

Comment:
Seems strange to deprecate with no explanation/replacement. If we don't want to support this constructor can we just remove it now, or just leave it un-deprecated?